JRuby 1.7.0-dev snapshot url broken

Hi,

the download url of the 1.7.0-dev snapsho has changed:

```
$rbenv install jruby-1.7.0-dev
Downloading http://ci.jruby.org/snapshots/jruby-bin-1.7.0.dev.tar.gz...

BUILD FAILED
```

see my pull request.

--- a/share/ruby-build/jruby-1.7.0-dev
+++ b/share/ruby-build/jruby-1.7.0-dev
@@ -1,1 +1,1 @@
-install_package "jruby-1.7.0.dev" "http://ci.jruby.org/snapshots/jruby-bin-1.7.0.dev.tar.gz" jruby
+install_package "jruby-1.7.0.dev" "http://ci.jruby.org/snapshots/1.7.x/jruby-bin-1.7.0.dev.tar.gz" jruby

